tts_engine.py

```
@@ -0,0 +1,103 @@
+        part = response.candidates[0].content.parts[0]
+        audio_bytes: bytes = part.inline_data.data
+        mime_type: str = part.inline_data.mime_type or "audio/wav"
+
+        if debug_output_path:
+            with open(debug_output_path, "wb") as fh:
+                fh.write(audio_bytes)
+            logger.debug("Debug audio written to %s", debug_output_path)
+
+        return audio_bytes, mime_type, None
+
+    except Exception as exc:  # pragma: no cover – runtime API errors
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)